internal/component/prometheus/remotewrite/types.go

```diff
@@ -242,6 +243,9 @@ func convertConfigs(cfg Arguments) (*config.Config, error) {
 			SendExemplars:        rw.SendExemplars,
 			SendNativeHistograms: rw.SendNativeHistograms,
 
+			//TODO: Make this configurable?
+			ProtobufMessage: config.RemoteWriteProtoMsgV1,
```

Comment:
This feels like we would need this when we upgrade to prom 3.0, so maybe add that to the todo?

Reply:
Ok, done. This is why I added the todo, but I should have been clear that it's related to v3.